libs/core/src/components/pds-button/docs/pds-button.mdx

```diff
@@ -175,10 +173,63 @@ The loading prop allows a `pds-button` to display a loading state.
   <pds-button variant="primary" loading="true">Loading</pds-button>
 </DocCanvas>
 
+## Icon Only
+
+The icon-only prop allows a `pds-button` to display an icon only.
+
+While the text is visually hidden, it is still remains accessible.
+
+> ***Note:*** An icon is required to be provided.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+    <PdsButton variant="primary" iconOnly="true" icon="favorite">Icon Only</PdsButton>
+    <PdsButton variant="secondary" iconOnly="true" icon="favorite">Icon Only</PdsButton>
+    <PdsButton variant="accent" iconOnly="true" icon="favorite">Icon Only</PdsButton>
+    <PdsButton variant="destructive" iconOnly="true" icon="favorite">Icon Only</PdsButton>
+    `,
+    webComponent: `
+    <pds-button variant="primary" icon-only="true" icon="favorite">Icon Only</pds-button>
+    <pds-button variant="secondary" icon-only="true" icon="favorite">Icon Only</pds-button>
+    <pds-button variant="accent" icon-only="true" icon="favorite">Icon Only</pds-button>
+    <pds-button variant="destructive" icon-only="true" icon="favorite">Icon Only</pds-button>
+    `
+}}>
+  <pds-button variant="primary" icon-only="true" icon="favorite">Icon Only</pds-button>
+  <pds-button variant="secondary" icon-only="true" icon="favorite">Icon Only</pds-button>
+  <pds-button variant="accent" icon-only="true" icon="favorite">Icon Only</pds-button>
+  <pds-button variant="destructive" icon-only="true" icon="favorite">Icon Only</pds-button>
+</DocCanvas>
+
+
+## CTA Button
```

Comment:
This naming is weird as not every link is a call to action. We're making it a link under the hood, so that part should be explicitly stated since that's what they are looking for, even in the naming since devs mostly skip the docs titles. Here me out... `Link Button`. It says what it does

Reply:
@Kajabi/dss-devs any additional thoughts here. I don't fully disagree and looking at other systems there are quite a few that have a component called Button Link.

Reply:
Adding my 2 cents: I'm leaning towards naming this component 'Button Link' or 'Link Button' to keep its purpose clear and explicit.

Reply:
I agree with Button Link or Link Button

Reply:
@Kajabi/dss-devs I have updated the related docs with the change based on everyone's comments. thanks all!